paddle/framework/channel_test.cc

```diff
@@ -72,4 +72,5 @@ TEST(Channel, ConcurrentSendNonConcurrentReceiveWithSufficientBufferSize) {
   std::this_thread::sleep_for(std::chrono::milliseconds(100));  // wait 0.5 sec
   EXPECT_EQ(sum, 45U);
   CloseChannel(ch);
+  t.join();
```

Comment:
It seems that you closed but didn't delete the channel in this test?  In my mind, this test should look like

```c++
std::unique_ptr<Channel<int>*> ch(MakeChannel<int>(10)); // `delete ch.get()` will be called
do_something_here_with(ch);
CloseChannel(ch.get());
```

Reply:
> It seems that you closed but didn't delete the channel in this test?

We can add `DeleteChannel` behind `CloseChannel`.

Reply:
Maye the use of unique_ptr didn't present my idea clearly. Let us use `delete` explicitly:

```c++
auto ch = MakeChannel<int>(10);
do_something(ch);
CloseChannel(ch);
delete ch;
```

Reply:
Ok!
If we use `delete` to destroy the `ch` but other threads also hold the `ch` and these threads send data to the `ch`, there will cause an error. 
I wonder that whether this case is existent.